script.js

```diff
@@ -130,6 +130,24 @@ document.getElementById('contact-form').addEventListener('submit', (e) => {
   }
 });
 
+/* Local Storage */
+function saveData(){
+
+  let nameLocal = document.getElementById("full-name").value;
+  let emailLocal = document.getElementById("email-address").value;
+  let bodyLocal = document.getElementById("message").value;
+
+  const pData = {
+    "name" : nameLocal,
+    "email" : emailLocal,
+    "bodyLocal" : bodyLocal,
+  }
+
+  window.localStorage.setItem("pData", JSON.stringify(pData));
+}
+
+
+
```

Comment:
- [ ] There're some project requirements you need to implement here before you project can be due for approval.
- After filling the form, and not submitting it, the entered values should be saved in the browser. This behaviour is failing in your project. Kindly look into it and fix. Take a look :point_down: 

### Your Project Behaviour.


https://user-images.githubusercontent.com/26597432/202740777-41b9cf27-5145-4b4b-8c39-5029f9cc33a0.mp4

- [ ] When visiting the page the form inputs should be contained, be auto-populated with, the values entered in the previous visit to the page. After filling the form and clicked submit button, I was taken to the formspree. I came back to your page and reloaded the page but the previously inputed data got disappeared. This should not be so, the values on your form should remain there on page load. Take a look. :point_down: 

### Your Form Behaviour


https://user-images.githubusercontent.com/26597432/202741732-d348efe1-43f6-4e86-a6e4-c9ecc1243b8d.mp4